oci/table_oci_advanced_resource_query_search.go

```diff
@@ -0,0 +1,226 @@
+	return &plugin.Table{
+		Name:        "oci_advanced_resource_query_search",
+		Description: "OCI Advanced Resource Query Search",
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.SingleColumn("query"),
+			Hydrate:    listAdvancedResourceQuerySearch,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "identifier",
+				Description: "The unique identifier for this particular resource, usually an OCID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "display_name",
+				Description: "The display name (or name) of this resource, if one exists.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "availability_domain",
+				Description: "The availability domain where this resource exists, if applicable.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "resource_type",
+				Description: "The resource type name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The lifecycle state of this resource, if applicable.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "The time that this resource was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+			{
+				Name:        "query",
+				Description: "The query based on which the search was done.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "identity_context",
+				Description: "Additional identifiers to use together in a Get request for a specified resource, only required for resource types that explicitly cannot be retrieved by using a single identifier, such as the resource's OCID.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "search_context",
+				Description: "SearchContext Contains search context, such as highlighting, for found resources.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// tags
+			{
+				Name:        "defined_tags",
+				Description: ColumnDescriptionDefinedTags,
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "freeform_tags",
+				Description: ColumnDescriptionFreefromTags,
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "system_tags",
+				Description: "System tags associated with this resource.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(resourceQuerySearchTags),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "search_region",
```

Comment:
Is it common to change the standard `region` column name? If not, is there a reason we changed it here?

Reply:
Confirmed with @bigdatasourav that this is by design per his conversation with @LalitTurbot